model_analyzer/config/generate/undirected_run_config_generator.py

```diff
@@ -197,44 +192,50 @@ def _get_magnitude(self):
         return self._search_config.get_step_magnitude() + self._magnitude_offset
 
     def _get_next_run_config(self):
-        # TODO multi-model
-        mc = self._get_next_model_config()
+        run_config = RunConfig(self._triton_env)
 
-        model_variant_name = mc.get_field('name')
-        pac = self._get_next_perf_analyzer_config(model_variant_name)
+        for i, _ in enumerate(self._models):
+            mrc = self._get_next_model_run_config(i)
+            run_config.add_model_run_config(mrc)
 
-        mrc = ModelRunConfig(self._models[0].model_name(), mc, pac)
+        return run_config
 
-        run_config = RunConfig(self._triton_env)
-        run_config.add_model_run_config(mrc)
+    def _get_next_model_run_config(self, model_num):
+        mc = self._get_next_model_config(model_num)
 
-        return run_config
+        model_variant_name = mc.get_field('name')
+        pac = self._get_next_perf_analyzer_config(model_variant_name, model_num)
 
-    def _get_next_model_config(self):
-        dimension_values = self._get_dimension_values(
-            self._coordinate_to_measure)
+        model_name = self._models[model_num].model_name()
+        return ModelRunConfig(model_name, mc, pac)
+
+    def _get_next_model_config(self, model_num):
+        dimension_values = self._get_coordinate_values(
+            self._coordinate_to_measure, model_num)
 
         param_combo = {
             'dynamic_batching': {},
             'max_batch_size':
                 dimension_values['max_batch_size'],
             'instance_group': [{
                 'count': dimension_values['instance_count'],
-                'kind': "KIND_GPU"
+                'kind': "KIND_GPU",
+                'rate_limiter': {
```

Comment:
What is this for?

Reply:
https://github.com/triton-inference-server/server/blob/aac903a7f727a186877b7c70cf18c460b0c8b906/docs/rate_limiter.md

In the future we will experiment with different rate limiter values for multi-model, and will likely search across those values. For now, while we are bringing up the basic algorithm for multi-model, we just want to guarantee even prioritization across models so that we don't end up with one model starved.